### DISCUSSION TITLE
Installing transformer Flair or TNER models from Huggingface

### DISCUSSION THREAD

**Discussion by emiltj** [explosion/spaCy#12230] [Help: Installation] [answered]
Hi,

I was looking for a way to use a way to use a transformer model from huggingface for NER tagging (more specifically [one from flair](https://huggingface.co/flair/ner-english-ontonotes-large) or one from [TNER](https://huggingface.co/tner/roberta-large-ontonotes5)).

Trying
`python -m spacy download flair/ner-english-ontonotes-large`
gives
```
_No compatible package found for 'flair/ner-english-ontonotes-large'
(spaCy v3.5.0)_
```

Is it possible to use the NER tagging from such a model in spaCy, perhaps using spacy-transformers?
And if not, is it possible to convert the output from Flair or TNER to Doc objects?

**Comment by emiltj** [comment, marked as answer]
I may have found a workaround, using spaCy wrap:

```
import spacy
import spacy_wrap

nlp = spacy.blank("da")
config = {"model": {"name": "tner/roberta-large-ontonotes5"}}
nlp.add_pipe("token_classification_transformer", config=config)
doc = nlp("Some text for NER")
```

**Reply by rmitsch** [reply]
Hi @emiltj, great you figured it out! [spaCy-wrap](https://spacy.io/universe/project/spacy-wrap/) seems like a reasonable way to do this.
